mailu/values.yaml

```diff
@@ -61,9 +61,24 @@ mail:
 
 # certmanager settings
 certmanager:
+  enabled: false
+  # leave emty for default value
+  #apiVersion: cert-manager.io/v1alpha3
+  apiVersion:
+  # leave emty for default value. If set one, make sure to set the same for ingress.secretName
+  #secretName: &secretName wildcard-cert-secret
+  secretName:
```

Comment:
secret name should be removed here and in ingress. There should be a setting "certificateSecretName" or so which is used at all places where a certificate secret is used:
* in ingress (if enabled)
* in certmanager certificate (if enabled)
* in front (for ssl termination).
I can see no point in making this individually configurable because it needs to be the same.